GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -82,8 +82,6 @@ public override void Execute()
         {
             int exitCode = 0;
 
-            // TODO(Mac): limit the length of the enlistment root based on the length constraints imposed by named pipes

```

Comment:
This comment was written back when we let .NET create the socket file in its default location, which was rather deep inside some folder structure, and therefore limited the length of the repo root.

We now place the socket file inside the `.gvfs` folder, so there is no longer any such restriction.